enhancements/monitoring/alerts-ui-management.md

```diff
@@ -0,0 +1,337 @@
+---
+title: alerts-ui-managment
+authors:
+  - "@sradco"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+api-approvers:
+  - TBD
+creation-date: 2025-07-24
+last-updated: 2025-07-30
+tracking-link:
+  - "https://issues.redhat.com/browse/CNV-46597"
+  - "https://issues.redhat.com/browse/CNV-62972"
+---
+# New APIs to support Alerts UI Management in OpenShift
+
+## Summary
+Provide a user‑friendly UI and REST API for defining, viewing, editing, disabling/enabling and silencing Prometheus alerts without manual YAML edits, reducing alert fatigue and improving operational efficiency.
+Platform alerts will be updated via entries in the `AlertRelabelConfig` CR (in `cluster-monitoring-config`) rather than editing their original `PrometheusRule`.
+
+## Motivation
+- The current `PrometheusRule` CRD + `oc apply` workflow is powerful yet cumbersome for day‑to‑day alert tweaks.
+- Some operational teams prefer an interactive console and API to manage alerts safely, guided by best practices.
+- A unified interface will help users create, clone, disable or silence alerts, view real‑time firing status and preserve changes across upgrades.
+
+### User Stories
+
+1. **Bulk disable during maintenance**
+   As a cluster administrator, I want to select and disable multiple alerts in one action,
+   so that I can suppress non‑critical notifications across all namespaces during planned maintenance windows.
+
+2. **Single‑alert toggle from alert details**
+   As an SRE on call, I want to click “Disable” or “Enable” on an individual alert’s details page,
+   so that I can quickly mute a noisy alert or restore it once the underlying issue is resolved.
+
+3. **Clone and customize platform alerts**
+   As a cluster admin, I want to clone an existing built‑in alert, adjust its threshold and severity, and save it as a user‑defined rule,
+   so that I can tailor default monitoring to suit my team’s SLAs without modifying upstream operators.
+
+4. **Create alert from a metric query**
+   As a developer, I want to write or paste a PromQL expression in a “Create Alert” form, specify duration and severity, and save it,
```

Comment:
What if you could create an alerting rule from the Metrics > Observe page?

Reply:
Not in scope of this effort, but possible later on I assume.